docs/technical-reference/api/README.md

```diff
@@ -1,15 +1,37 @@
 
-Additionally, we maintain the [Watcher custom resource](v1beta1/watcher_types.go) to define callback functionality for synchronized remote clusters that allows lower latencies before changes are detected by the control-plane.
+1. The introduction of a single entry point CustomResourceDefinition to control a cluster and it's desired state: [Kyma custom resource(CR)](../../../api/v1beta2/kyma_types.go).
+2. The introduction of a single entry point CustomResourceDefinition to control a module and it's desired state: [Manifest CR](../../../api/v1beta2/manifest_types.go)
+3. The [ModuleTemplate CR](../../../api/v1beta2/moduletemplate_types.go) which contains all reference data for the modules to be installed correctly. It is a standardized desired state for a module available in a given release channel.
 
-Read more about the custom resource defininitions in the respective documents:
+Additionally, we maintain the [Watcher custom resource](../../../api/v1beta2/watcher_types.go) to define the callback functionality for synchronized remote clusters that allows lower latencies before changes are detected by the Control Plane.
+
+## Custom Resource Definitions
+
+Read more about the custom resource definitions (CRDs) in the respective documents:
 
 - [Kyma CR](kyma-cr.md)
 - [Manifest CR](manifest-cr.md)
 - [ModuleTemplate CR](moduleTemplate-cr.md)
+
+## Synchronization of Module Catalog with remote clusters
+
+Lifecycle Manager ensures that the Module Catalog is correctly synchronized with users' runtimes.
+The Module Catalog consists of all modules that are available for a user. The Module Catalog portfolio may vary for different users.
+The synchronization mechanism described below is essential to allow users to enable modules in their clusters.
+The mechanism is controlled by the set of flags that are configured on Kyma and ModuleTemplate CRs in the Control Plane. The flags are: `operator.kyma-project.io/sync`, `operator.kyma-project.io/internal`, and `operator.kyma-project.io/beta`.
+The v1beta2 API introduces three groups of modules:
+
+- Standard modules, synchronized by default.
+- Internal modules, synchronized per-cluster only if configured explicitly on the corresponding Kyma CRs. To mark a ModuleTemplate CR as `internal`, use the `operator.kyma-project.io/internal` label and set it to `true`.
+- Beta modules, synchronized per-cluster only if configured explicitly on the corresponding Kyma CRs. To mark a ModuleTemplate CR as `beta`, use the `operator.kyma-project.io/beta` label and set it to `true`.
+
+By default, without any flags configured on Kyma and ModuleTemplate CRs, the `standard` ModuleTemplate CR is synchronized with the remote cluster.
+
+**NOTE:** Disabling synchronization for already synchronized ModuleTemplates CRs does not remove them from remote clusters. The CRs remain as they are, but any subsequent changes to these ModuleTemplate CRs in the Control Plane are not synchronized.
+
```

Comment:
I would add here another note. It's kind of redundant, because it's mentioned elsewhere, but this is the place where we describe the ModuleTemplate sync mechanism... Up to you.
"Note 2: the ModuleTemplate sync is enabled only when the LifecycleManager is started with `inKcpMode=true` flag. Otherwise it runs in a single-cluster mode, thus there's no need to sync any CRs"